docs/updating_css_json.md

```diff
@@ -0,0 +1,374 @@
+- Media it belongs to
+- Computed value
+- Animation type
+- Canonical order of values
+- Whether it is a shorthand for other properties
+- MDN groups it belongs to
+
+A fundamental element is that we document what is in the _specification_.
+That means the reference page is based on the latest iteration of that specification and later on there are compatibility notes stressing the differences between different specification versions and their different implementations by rendering engines.
+
+### Get information from the specifications
+
+Once you have found the relevant specification, the first thing to do is to look for a property definition in the specification.
+Look for the summary table (near the bottom), it has most of the values that you are looking for:
+
+![The summary table lists useful attributes of the CSS properties defined in the give spec.](./assets/property_index.png)
+
+In some specifications, especially in older drafts, there isn't such a property index table.
+In this case you need to look for the property box of the property itself which looks like this:
+
+![This box lists the most useful attributes of a CSS property.](./assets/property_definition.png)
+
+Note that in some cases the property index table might be more briefly and with less details than the property box.
+
+### Store the information in the JSONs
+
+The information from the specifications is stored in a specific way in the JSON DB. CSS properties are stored within [properties.json](../css/properties.json).
+All the JSON keys below are obligatory.
+
+Translated information is stored as key string mapping to a translation in [l10n/css.json](../l10n/css.json) with language + country codes (according to [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) and [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)) as keys.
+
+You can use the macros [`{{cssxref}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/cssxref.ejs) and [`{{SVGElement}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/SVGElement.ejs) within translations to refer to other parts of MDN.
+
+<table>
+  <tbody>
+    <tr>
+      <th>Spec. name</th>
+      <th>JSON key name</th>
+      <th>Comment</th>
+      <th>Examples</th>
+    </tr>
+    <tr>
+      <td>Value</td>
+      <td><code>syntax</code></td>
+      <td>As specified within the specification</td>
+      <td>
+        <pre>"syntax": "visible | hidden"</pre>
+      </td>
+    </tr>
+    <tr>
+      <td>Media</td>
+      <td><code>media</code></td>
+      <td>Either a predefined localization key, a block of localizations</td>
+      <td>
+        <details>
+          <summary>properties.json</summary>
+          <pre>"media": "visual"</pre>
+        </details>
+        <details>
+          <summary>l10n/css.json</summary>
+          <pre>
+"visual": {
+  "en-US": "visual",
+  "de": "visuell",
+  "fr": "visuel",
+  "ru": "визуальный"
+}</pre
+            >
```

Comment:
Probably a good idea to tidy the end pre up 👍

Reply:
Good, done in 38f0ca056bda795605f1875cc71e86d852016335